Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript

```diff
@@ -324,8 +324,8 @@ private command _CreateMobileScroller
          (_ControlType() is "table" and \
          the dgProps["show hscrollbar"] of me is not false) then
       if sScrollerId is empty or sScrollerId is not among the lines of mobileControls() then
-         mobileControlCreate "scroller"
-         put the result into sScrollerId
+         put the short name of me into sScrollerId
+         mobileControlCreate "scroller", sScrollerId
```

Comment:
@livecodepanos Using the name as the scroller ID here will cause issues if there are two open data grids with the same name. It is unlikely, but I guess you could use the long id of the control. Or maybe check to see if the name is not already in `mobileControls()`

Reply:
Thanks @livecodemichael, good point.

I have updated the title of the PR as well